### PR TITLE
show current page in primary nav

### PR DESCRIPTION
**Pull request by sarahframe**

### CODE REVIEW

File: cdhweb/pages/templatetags/core_tags.py [hunk 1]
```diff
@@ -187,4 +200,14 @@ def site_alerts(context):
         .exclude(display_until__lt=now)
     )
     data = {"site_alerts": site_alerts, "request": context.get("request")}
-    return data
\ No newline at end of file
+    return data
+
+
+@register.filter
+def starts_with(value, arg):
```
Comment (haydngreatnews) [inline cdhweb/pages/templatetags/core_tags.py:207]: Wild that this isn't a default tag (given some of the things that are)

File: cdhweb/pages/templatetags/core_tags.py [hunk 2]
```diff
@@ -77,14 +82,22 @@ def _get_primary_nav_items():
...
+def primary_nav_dict(context):
     """
     Return the primary navigation data as a dict, for use with the 'json_script' filter.
     """
     primary_nav_items = _get_primary_nav_items()
     l1_menu_item_data = [_l1_item_to_dict(item) for item in primary_nav_items]
 
+    current_path = context["request"].path
+
+    for item in l1_menu_item_data:
```
Comment (haydngreatnews) [inline cdhweb/pages/templatetags/core_tags.py:95]: Do we still need to do this, given we've used a different approach in the template?

Comment (liamjohnston) [reply cdhweb/pages/templatetags/core_tags.py:95]: If you're asking whether we need it in both the template and the JSON blob, then the answer is yes :/

Comment (haydngreatnews) [reply cdhweb/pages/templatetags/core_tags.py:95]: Ah, right. In my head we were using the same data for both

**Review by haydngreatnews — APPROVED** [review]

**Pull request merged**